src/main/java/gregtech/api/recipes/ModHandler.java

```diff
@@ -260,6 +260,10 @@ public static void addShapedRecipe(String regName, ItemStack result, Object... r
     }
 
     public static void addShapedEnergyTransferRecipe(String regName, ItemStack result, Predicate<ItemStack> chargePredicate, boolean transferMaxCharge, Object... recipe) {
+        addShapedEnergyTransferRecipeWithOverride(regName, result, chargePredicate, true, transferMaxCharge, recipe);
+    }
+
+    public static void addShapedEnergyTransferRecipeWithOverride(String regName, ItemStack result, Predicate<ItemStack> chargePredicate, boolean overrideCharge, boolean transferMaxCharge, Object... recipe) {
```

Comment:
I am not fond of this name but I understand why you named this function differently.
But I am not getting any idea for better method name myself.

Reply:
If you think of a better name, let me know and I will change it

Reply:
Fair enough, let's go with current one.